app/helpers/date_range_helper.rb

```diff
@@ -35,11 +34,11 @@ def selected_range
     when "last_month"
       (now - 1.month).beginning_of_month..(now - 1.month).end_of_month
     when "this_year"
-      now.beginning_of_year..now
+      now.beginning_of_year..now.end_of_year
```

Comment:
@armahillo bug fix - otherwise "this_year" wouldn't show thing schedule to go, for example, tomorrow